recon/cli.py

```
@@ -0,0 +1,262 @@
+    name: str = typer.Argument(..., help="Playbook name (e.g., infra_quick)"),
+    arg: List[str] = typer.Option([], "-a", "--arg", help="Argument key=value, repeatable"),
+    json_out: bool = typer.Option(False, "--json", help="Print JSON result"),
+):
+    """Run a playbook."""
+    console: Console = ctx.obj["console"]
+    config = ctx.obj["config"]
+    no_cache: bool = ctx.obj["no_cache"]
+    cache_ttl: int = ctx.obj["cache_ttl"]
+
+    if name not in PLAYBOOKS:
+        raise typer.BadParameter(f"Unknown playbook: {name}")
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)